packages/frontend/src/components/common/Button.js

```diff
@@ -2,6 +2,26 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import styled from 'styled-components';
 
+const backgroundColorMap = {
```

Comment:
Nice mapping of colors! I'm curious why you decided to use this button component vs the regular `FormButton`? Seems it would be good if we used the same button as much as possible for consistency sake, and less components to maintain.

Reply:
Definitely. Will switch to the FormButton.